src/lib/cimbar_js/cimbar_recv_js.cpp

```diff
@@ -29,29 +29,18 @@ namespace {
 	TimeAccumulator _tImgDecode;
 
 	// settings
-	unsigned _colorBits = 2;
 	int _modeVal = 68;
```

Comment:
Trivia: the magic modeVal number originated in the android app (because I didn't want to use strings), and gradually worked its way here.

`6` => 0.6
`8` => 8x8 grid

(of course, the widescreen config is set at `67` so my perfect flawless numbering scheme has already failed...)